dora/src/baseline/cannon/codegen.rs

```diff
@@ -587,6 +597,55 @@ where
             .store_mem(bytecode_type.mode(), Mem::Local(offset), REG_RESULT.into());
     }
 
+    fn emit_jump_if(
+        &mut self,
+        bytecode: &BytecodeFunction,
+        src: Register,
+        bytecode_idx: &BytecodeIdx,
+        op: CondCode,
```

Comment:
`CondCode` allows all kinds of comparison operators, I guess it is better to just pass in `true` and `false` instead.